classes/migration/upgrade/PKPv3_4_0UpgradeMigration.inc.php

```diff
@@ -62,6 +62,12 @@ public function up()
             // No reviewer can be assigned twice on the same review round.
             $table->unique(['review_round_id', 'reviewer_id'], 'review_assignment_reviewer_round_unique');
         });
+
+        // TODO: See where this should go
+        // pkp/pkp-lib#7129: Convert the IssueService to the new Repository/DAO/Map pattern
+        Schema::table('issue_settings', function (Blueprint $table) {
```

Comment:
Is this upgrade file an okay place to include this change or would this need its own, individual migration file. Same with the following code in the `down()` method.

Reply:
Alec will know better where to draw the line with new migration files. But either way this needs to go into OJS, not pkp-lib.

Reply:
I would suggest creating a `classes/migration/upgrade/v3_4_0/I7129_EntityDAORefactor.inc.php` in the OJS repo. For a one-liner like this it's questionable making a whole class out of it, but I'd like to start moving towards the pattern of keeping a directory of small migrations for each version (rather than having omnibus classes) and it's likely there will be other changes required for pkp/pkp-lib#7129 that can go here too.